staging/src/k8s.io/component-base/logs/options.go

```diff
@@ -88,13 +90,19 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 
 	// No new log formats should be added after generation is of flag options
 	logRegistry.Freeze()
+	fs.BoolVar(&o.LogSanitization, "logging-sanitization", false, `Enables runtime sanitization of logs based on "datapolicy" struct tags.
```

Comment:
please prefix with `experimental-` ( like the others https://cs.k8s.io/?q=fs%5C..*%22experimental-&i=nope&files=&repos=kubernetes/kubernetes )

Also can we just drop the first sentence? that is just aimed at developers not end users.

And drop `that Kubernetes contributors` as well.

Reply:
done